src/src/modules/edit-requests/edit-requests.service.ts

```diff
@@ -127,12 +145,24 @@ export class EditRequestService {
 			});
 		}
 
-		let update_body = {
-			status: body?.status,
-		};
+		if (body?.status == 'approved' && body?.fields?.length == 0) {
+			return res.json({
+				status: 400,
+				message: 'INVALID_PARAMETERS',
+				data: {},
+			});
+		}
+
+		if (body?.status == 'closed') {
+			body.fields = '[]';
```

Comment:
`body.fields = '[]';` change to `body.fields = [];`

Reply:
resolved